File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/SaveCheckedOperationsReqVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;
import java.util.Set;

@Data
public class SaveCheckedOperationsReqVo implements Serializable {
    private String roleId;
    private String projectId;
    private Set<Long> checkedKeys;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/SaveCheckedOperationsReqVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable


@AllOpen
class SaveCheckedOperationsReqVo : Serializable {
    var roleId: String? = null
    var projectId: String? = null
    var checkedKeys: Set<Long>? = null
}
